logic/src/commonMain/kotlin/com/wire/kalium/logic/data/team/TeamRepository.kt

```diff
@@ -19,14 +20,16 @@ interface TeamRepository {
     suspend fun fetchTeamById(teamId: TeamId): Either<CoreFailure, Team>
     suspend fun fetchMembersByTeamId(teamId: TeamId, userDomain: String): Either<CoreFailure, Unit>
     suspend fun getTeam(teamId: TeamId): Flow<Team?>
+    suspend fun deleteConversation(conversationId: ConversationId, teamId: String): Either<CoreFailure, Unit>
 }
 
+@Suppress("LongParameterList")
```

Comment:
I think since you didn't change the parameters list, that won't be needed. right?